plugins/modules/aci_fabric_pod_remote_pool.py

```diff
@@ -0,0 +1,285 @@
+                    "ownerTag": ""
+                }
+            }
+        }
+    ]
+proposed:
+  description: The assembled configuration from the user-provided parameters
+  returned: info
+  type: dict
+  sample:
+    {
+        "fvTenant": {
+            "attributes": {
+                "descr": "Production environment",
+                "name": "production"
+            }
+        }
+    }
+filter_string:
+  description: The filter string used for the request
+  returned: failure or debug
+  type: str
+  sample: ?rsp-prop-include=config-only
+method:
+  description: The HTTP method used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: POST
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec, aci_owner_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(aci_owner_spec())
+    argument_spec.update(
+        podId=dict(type=int, aliases=["pod"]),
+        description=dict(type=str, aliases=["desc"]),
+        remoteId=dict(type=int, aliases=["id"]),
+        nameAlias=dict(type=str),
+        remotePool=dict(type=str, aliases=["pool"]),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["podId", "remoteId"]],
+            ["state", "present", ["podId", "remoteId"]],
+        ],
+    )
+
+    aci = ACIModule(module)
+
+    podId = module.params.get("podId")
+    description = module.params.get("description")
+    remoteId = module.params.get("remoteId")
+    nameAlias = module.params.get("nameAlias")
+    remotePool = module.params.get("remotePool")
+    state = module.params.get("state")
+
+    if podId is not None and int(podId) not in range(1, 254):
+        aci.fail_json(msg="Pod ID: {0} is invalid; it must be in the range of 1 to 254.".format(podId))
+
+    aci.construct_url(
+        root_class=dict(
+            aci_class="fabricSetupP",
+            aci_rn="controller/setuppol/setupp-{0}".format(podId),
+            module_object=podId,
+            target_filter={"podId": podId},
+        ),
+        subclass_1=dict(aci_class="fabricExtSetupP", aci_rn="extsetupp-{0}".format(remoteId), module_object=remoteId, target_filter={"extPoolId": remoteId}),

```

Comment:
```suggestion
        subclass_1=dict(
            aci_class="fabricExtSetupP",
            aci_rn="extsetupp-{0}".format(remoteId),
            module_object=remoteId,
            target_filter={"extPoolId": remoteId},
        ),

```

Reply:
I did run black format over the code and this is the format it gave me.
Would have prefered to format like you suggested instead.